saleor/graphql/translations/mutations.py

```diff
@@ -25,37 +31,23 @@
 
 
 TRANSLATABLE_CONTENT_TO_MODEL = {
-    str(
-        translation_types.ProductTranslatableContent
-    ): product_models.Product._meta.object_name,
-    str(
-        translation_types.CollectionTranslatableContent
-    ): product_models.Collection._meta.object_name,
-    str(
-        translation_types.CategoryTranslatableContent
-    ): product_models.Category._meta.object_name,
-    str(
-        translation_types.AttributeTranslatableContent
-    ): attribute_models.Attribute._meta.object_name,
-    str(
-        translation_types.AttributeValueTranslatableContent
-    ): attribute_models.AttributeValue._meta.object_name,
-    str(
-        translation_types.ProductVariantTranslatableContent
-    ): product_models.ProductVariant._meta.object_name,
-    str(translation_types.PageTranslatableContent): page_models.Page._meta.object_name,
-    str(
-        translation_types.ShippingMethodTranslatableContent
-    ): shipping_models.ShippingMethod._meta.object_name,
-    str(
-        translation_types.SaleTranslatableContent
-    ): discount_models.Sale._meta.object_name,
-    str(
-        translation_types.VoucherTranslatableContent
-    ): discount_models.Voucher._meta.object_name,
-    str(
-        translation_types.MenuItemTranslatableContent
-    ): menu_models.MenuItem._meta.object_name,
+    str(translation_types.ProductTranslatableContent): str(product_types.Product),
```

Comment:
This should map type names to model names. Currently in all cases the names are the same but I'm not sure if it always stay like that so I think it's safer to keep the mapping as it was before.

Reply:
@maarcingebala This mapping is being passed to `graphene.Node.to_global_id` which expects a graphene type 🤔 
<img width="837" alt="image" src="https://user-images.githubusercontent.com/2566928/135266259-af3cbffd-9843-42ee-908c-d65646dccdf6.png">

Reply:
You're right, it should map translatable content types to their corresponding "base" types, so `PageTranslatableContent` to `Page`. So the changes are correct and the mapping could be actually renamed to something like `TRANSLATABLE_CONTENT_TO_TYPE`.

Reply:
I've changed it to `TRANSLATABLE_CONTENT_TO_TYPE`